index.html

```diff
@@ -0,0 +1,119 @@
a5sXxrjNpLg9SfwMcmJSb5/wCCH0/9cbzaRiNArX5R1Lx34tVSQEk+4ocgebOhPqT693m4lRnqPBQO3HlGh5BPHnkgcMTVaQb1yZGFkYnirH/TrULvPkJGLFFQBjRarPBPk/z16kUmtmkeRJVKotns2y+sdhu4DhQ3DBDJHLUYZhiueKllJ9Io8kALYIB133+w2MCLJ3IldlVhagu11RRKJAoiygA5rjXhvUI/alAoD0rV1Rs8mzrnt+v7pDSzORiIgHqQBAQQoDggAEAgDxrKXwalynQ4fEODo9R6F09p5fzWcloScsmViBLAoIIIesSQD4oe+jPqv6W2UW2nyz7uEkiM0sjSHH9RFnlLxy4oEg/Gnn+FczbiN1bEdrBf0inpmAJH8P6TwPTz4+aZ/iP9RMN5jgKxljWjVBnkjfLzlkFHx5/e8IRyeSr4OjLkjL0PMH3MiSdwOcgxNkmwQ90w49+a++r79F7iLe/iF3Koe2sZjo4sqkUwAAbJRitn5Is8ivOiOP21NISEMgYghgorj73evRlHZUcEJ6s9l3H0l0pXSI92OSY4ouM6hiATeWKhgOPYedLvpzrPT9ruKjeVEAlVy7Pjkpx9IEjBh6g1+QBd1evPperyD8NIKziTEEqnI5ABpRdD3bI8+dJ2mJrnwPYn+v7/APxqFi4qTbHLJzwj0LrW5gk3MgjAkQlVDC6a5Wa/TxjbUCbJABPxpB1Hcy7SWdJFBdmdGDclSJLJ5Hk1+r3DfvpNspGCtR4445/+dE9X2x7fdZyzGRkJPk0AbJJJJ5OtUqE5OS+gyPVY3iqvU1EjnimJPihXnj/2/fVw+j/8T5Nu6LuHaaABYwpSPJFBADA8WQo8Mx8+eOfJ1YjTrZ7QMgayPGk9UuQTlJnq/wBZQbOCZN9tU3LM8jSXkkG3VlNtiWQMWyIbiwQH5oarn+I2/jm3PeU2rrE5Cm6PaAIB+QQR59tUrdv7ewrgXX71fk+SdOf/AE7KHZqGrvyyRk1+kr2qPkX/AJp4/wBvnnRBLZNhJ8NDX6ZUK5Cl8WiZkZlKsQSR4Um6YMtiwSnHtq2dRAXbzvfqBq/DAl0AIJHBFhhfHAPI8+f9A3Ha3X4cC7DoW+RRY2K9xx5/+NXvq/07AYpdw2Zfg8O6rxQHpBrx86wzNeQ6sb/R+9FE320kmP5spks+GmeQ2T5rL5/4PjXCL6bLqHUooPs2ZI5PmjV+NXfdqzQwyMxJcLdtI3OJPu5HkfHvofpMQ7K0AKB4HjyfH21zT+InCPB24/hMc5VX84KFvNuImZGx4+AoB4+4/fV73XQpYJdts5n2S5KZBI8I7rG0bt5MWBc5ABeAar35p31Cn/8AYkFnkn3NfHgH7atUP1fPvJ9tDKzMs00LDIoTFg+P5WMagEi7sEHjjjnot6p/I4JUptL3O+z+km5BkjQC6KKED0eMwFuyeB6q+9aHG46cm7jmnw3XDxtGtM14tjwxAvIiueKAvSLqHVvwW5nESsG7kgZlkYA4MU/SbU3RIzDVkftov6w6aIj+LjODo61Q88iibJs39qPiq1SfKv1FKNp16CrcNlusFj7TNKiGNCVxyIFCy2J582aOrN0rbxwNHBOM1ChT4pcpG8+kNfimBv1N54IpHVOuzTbgblsFlsNlGoT1BiymvFjgfsou9WHoUxlRWcsWsAliWJOZNknn+t6ea1EMDTmT6x0Yt3cIXCIFYM6yFWoNme43sBRoHmvGunQ/pyopzKqgrG7JiciLjksNYPBxHF35+dWKePvB4mLYqBx3JCvAsUuWI5+x0Ym2qKZibDJQHqsUst+WI5v2A/nrlWWVKL+X7HZLDFOU0ur/AHPP4ehDsDcMxCuHIVVj4xP++VSb+FDH+w1YJZlh2uzd5XUdtjSyBA1stFgJFd6+FV+Cb12YiPZbSGie7C/IIAH6PIqz+r5A49/a69J6ZF2YY3USKdstq3KMD5tT7+4N8avJNqnL3f7mMEknr6pfsyo9W+rFkL1DuCLa/SuDEr5KKQSt8/xEAHxdCs9P2z7xkB3T5cIFs2q8KMaBpfV+373r0Yf4d7aFhuYpZ0vtnth0KWzov8aMSPVdEk8eR7VXcfS6bOeQq5Zvwu6lUi0xaJfFA8qfj/8AeqhmxviJg4Sd2WHp8O0G8/Auu33cTU6u4R3/AMsv6WWvgDjj+p1mx+n4rQqsasx8Jttv/qANMQSou/k/fjW/8N+hxz+ub1OoBDABTxHx49wPcV76usyCPsMvl54o+bICs+Te/mhj9rv2rXJmlPeos6YOEI/mXNAX/pmMiooTER16o8mruE1fbPFeBfB5oVelH1l0pn/CAm0A3K+Kti6YAWaHpVvJHj+l43mxilouit6VNMqt7WPI1X/8T+tr0/bpue13SSYgpfFRmGaz6TdV4FfvrHHGUpUu0CzpOLZUN70yBGSTcJHHEgxyYEg+1uppbN1xwaHnTTbdU2zxhEkjZxEthJAWDB6OaqbPx7gWPOlC7vd9V2kM3fXbgGS44oziWVmAIJewP9pJF86o3SfqF9tPKGBktHjyDtG2QJxdiLyIrwfPHPGumGJtNN20aSz9OqTPSP8AD76d7uw27jG+7yCqkgK9cekkGvuPA++o/XP0/L+P2zIzGMHb5ooyFCVizMKPAX3IPkcc6Q9I/wAR96qfk9pRdESJnZ5JIxwA9h49r86YxfX88cRkaNJpsmDSSXRBc2FVaxGIxoH2U81RJQnGWy75I2crj3EsG5SLMCKCKU05iVhgCb5X9Cqoq28D9J44vVV+vPxUMDCeKCOLNY1RGyXLLP0+gFAygrkCCK9rJ0H1n/EmSSbbPHt4okjOZjssHJPORAHFeBXnnnxpd9aF9xBt9w7t+Zn+WAqxpicTiFUeSMufGRrTxxcZJyJm201FFS2XUCqBTHnXg2w4+ONZrru+mhGxu+FN1XkA/Os127pnLrJcH//Z" alt="picture of the city" height="90px">
```

Comment:
Is there a better way to implement the picture? this is a lot of noise code. ;-)

Reply:
I know 🙈 I just added everything locally now :)